doc/development.md

```diff
@@ -28,24 +28,23 @@ Install additional packages for Fedora, CentOS, and RHEL:
 
     sudo dnf install -y `cat automation/check-patch.packages`
 
-Create virtual environment for vdsm:
-
-    make venv
-
 
 ## Building Vdsm
 
-To configure sources (run `./configure --help` to see configuration options):
+To prepare sources (create Makefiles and configure script):
 
-    git clean -xfd
     ./autogen.sh --system --enable-timestamp
-    make
 
-To test Vdsm (refer to tests/README for further tests information):
```

Comment:
With this, we lose the reference to the tests/README, which is not mentioned either in the main README.
And we mix creating a virtual environment, which is meant for testing mainly, with the `Building Vdsm` section.

Following Nir's suggestion here: https://github.com/oVirt/vdsm/pull/255#discussion_r898273342
I think it would be nice to have a `Testing Vdsm` section (or similar name) with very basic, everyday commands (e.g., make venv once, then activate, a simple test run with tox, and deactivate, that's all), and refer to the tests/README for further information.

Reply:
I agree. But it probably shouldn't be the storage tests then, for the simple example, because those are more complicated.

Reply:
There's also a lot of testing-related make-targets, and it's not obvious, which ones should actually be used when, I think.

Reply:
AFAIK the testing make-targets are meant mostly for the pipelines, developers will rely more on tox (or pytest) directly, since we run only what tests our change set.
In any case, I would not put them here.

Reply:
Ok, I didn't add anything about the make targets, I just left in the ones that were already described.